### PR TITLE
[Fix/aut1035] - Read the Docs maintenance

### PR DESCRIPTION
**Pull request by ghost**
**Objective**
- Fixing the Build status badge.
- Fixed SmartPeakCLI documentation: Example code snippets weren't shown.
- Added a sub-guide for `smartpeak-cli` container usage.

### CODE REVIEW

**Review by dmccloskey — COMMENTED** [review]
Glad to see the badge URL has been updated.  I am not able to find the documentation for the CLI on the ReadTheDocs.  Am I just missing it?

**Comment by ghost** [comment]
> Glad to see the badge URL has been updated. I am not able to find the documentation for the CLI on the ReadTheDocs. Am I just missing it?

It can be found under the `SmartPeak Command Line Tool` of the `Step-by-step guides`. Do you prefer if we move it elsewhere?

**Comment by dmccloskey** [comment]
I am not seeing it there for some reason
StepByStep

**Comment by ghost** [comment]
> I am not seeing it there for some reason StepByStep

You are viewing the docs built for the `develop` branch in which the modifications made here hasn't reached yet, it can be viewed here in this branch : https://smartpeak-.readthedocs.io/en/fix-aut1035/

**Comment by dmccloskey** [comment]
>Got it.
>
>Under the section for "Using SmartPeak", can you add in a section for the GUI and another for the CLI?  Or alternatively, a Section for "Using SmartPeak", "Using the SmartPeak GUI", and "Using the SmartPeak CLI"?

I could rename "Using SmartPeak" to "Using SmartPeak GUI" and "SmartPeak Command-line Tool" to Using "SmartPeak CLI" then add both as subsection of "Using SmartPeak" which will have a brief introduction about the available usage options. Do you think this is an easy to follow structure?

**Comment by dmccloskey** [comment]
> > Got it.
> > Under the section for "Using SmartPeak", can you add in a section for the GUI and another for the CLI?  Or alternatively, a Section for "Using SmartPeak", "Using the SmartPeak GUI", and "Using the SmartPeak CLI"?
> 
> I could rename "Using SmartPeak" to "Using SmartPeak GUI" and "SmartPeak Command-line Tool" to Using "SmartPeak CLI" then add both as subsection of "Using SmartPeak" which will have a brief introduction about the available usage options. Do you think this is an easy to follow structure?

Sounds good to me.  Please go ahead with that.

**Comment by ghost** [comment]
> > > Got it.
> > > Under the section for "Using SmartPeak", can you add in a section for the GUI and another for the CLI?  Or alternatively, a Section for "Using SmartPeak", "Using the SmartPeak GUI", and "Using the SmartPeak CLI"?
> > 
> > 
> > I could rename "Using SmartPeak" to "Using SmartPeak GUI" and "SmartPeak Command-line Tool" to Using "SmartPeak CLI" then add both as subsection of "Using SmartPeak" which will have a brief introduction about the available usage options. Do you think this is an easy to follow structure?
> 
> Sounds good to me. Please go ahead with that.

The new structure can be viewed here: https://smartpeak-.readthedocs.io/en/fix-aut1035/ Please let me know if you have any further suggestions.

File: dockerfiles/Dockerfile.smartpeak-cli
Comment (dmccloskey) [inline dockerfiles/Dockerfile.smartpeak-cli:20]: Make sure my name is on the maintainer for all of the source files.

**Review by dmccloskey — COMMENTED** [review]
Looks very good.  One small comment to take care of and once the CI passes we should be good to merge.  Nice work!

File: dockerfiles/Dockerfile.smartpeak-cli
Comment (ghost) [reply dockerfiles/Dockerfile.smartpeak-cli:20]: Sure thing.

Comment (ghost) [reply dockerfiles/Dockerfile.smartpeak-cli:20]: Should be good to go now.

**Review by dmccloskey — APPROVED** [review]

**Pull request merged**
